src/pages/test-results-search/components/advanced-search/advanced-search.html

```diff
@@ -0,0 +1,57 @@
+<ion-grid>
+  <ion-row>
+    <ion-col col-46>
+      <input
+        placeholder="DTC number"
+        type="text"
+        id="dtc-number"
+        class="dtc-number"
+        [class.ng-invalid]="invalid"
+        [value]="dtcNumber"
+        (change)="dtcNumberChanged($event.target.value)">
+    </ion-col>
+    <ion-col col-4></ion-col>
+    <ion-col col-46>
+      <input
+        placeholder="DE or staff ID number"
+        type="text"
+        id="staff-number"
+        class="staff-number"
+        [class.ng-invalid]="invalid"
+        [value]="staffNumber"
+        (change)="staffNumberChanged($event.target.value)">
+    </ion-col>
+  </ion-row>
+  <ion-row>
+    <hr />
+  </ion-row>
+  <ion-row>
+    <ion-col class="component-label" col-22>
+      <label>Date range</label>
+    </ion-col>
+    <ion-col>
+      <ion-item>
+        <ion-label>from:</ion-label>
+        <ion-datetime
+          displayFormat="DD/MM/YYYY"
+          placeholder="select a date"
+          (ionChange)="startDateChanged($event)"></ion-datetime>
+      </ion-item>
+    </ion-col>
+    <ion-col>
+      <ion-item>
+        <ion-label>to:</ion-label>
+        <ion-datetime
+          displayFormat="DD/MM/YYYY"
+          placeholder="select a date"
+          (ionChange)="endDateChanged($event)"></ion-datetime>
+      </ion-item>
+    </ion-col>
+  </ion-row>
+  <ion-row>
+    <button [attr.disabled]="showSpinner ? 'true' : null" class="mes-primary-button search-tests" ion-button (click)="searchTests()">
+      <span class="search-tests-text">Search tests</span>
```

Comment:
The buttons in the app normally wrap a `<h3>` element to style them as bold.

Reply:
Changed it in both tabs